addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6727,6 +6727,10 @@ registry.ImageTools = ImageHandlerOption.extend({
             }
         }
         await this._reapplyCurrentShape();
+        // When the hover effects are first activated from the "animationMode"
+        // function of the "WebsiteAnimate" class, the history was paused to
+        // avoid recording intermediate steps. That's why we unpause it here.
+        this.options.wysiwyg.odooEditor.historyUnpauseSteps();
```

Comment:
A last detail to fix about this: here you unpause every time you set a hover effect while it was paused only the first time when choosing "onHover" for the animation mode
So the rest of the time, you unpause something that was not paused before :thinking: 

 So what I suggest is to add a check before unpausing:
- In the `notify` function when it is called with `enable_hover_effect` set a variable in `this` to `true` (something like `this.firstHoverEffect` or `this.hoverPauseStep`) (see "HERE" comment :arrow_down:)
  
  ```js
   /**
     * @override
     */
    notify(name) {
        if (name === "enable_hover_effect") {
            this.trigger_up("snippet_edition_request", {exec: () => {
                ...
                // Add the "Overlay" hover effect to the shape.
                this.firstHoverEffect = true; // <-- HERE
                const hoverEffectOverlayWidget = this._requestUserValueWidgets("hover_effect_overlay_opt")[0];
                hoverEffectOverlayWidget.enable();
                hoverEffectOverlayWidget.getParent().close(); // FIXME remove this ugly hack asap
            }});
        ...
  ```
- And then you check for it here before unpausing and delete it afterwards
```js
// When the hover effects are first activated from the "animationMode"
// function of the "WebsiteAnimate" class, the history was paused to
// avoid recording intermediate steps. That's why we unpause it here.
if (this.firstHoverEffect) {
    this.options.wysiwyg.odooEditor.historyUnpauseSteps();
    delete this.firstHoverEffect;
}
```

Like this we guarantee to not unpause something that shouldn't :wink: 
(I suggest here because the only time it passes through `notify` is when we activate the option for the first time :slightly_smiling_face: )

Reply:
ok good idea, I made the change :+1: